travis/h2o_integration_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -x
+
+# https://stackoverflow.com/questions/42218009/how-to-tell-if-any-command-in-bash-script-failed-non-zero-exit-status/42219754#42219754
+# Set err to 1 if pytest failed.
+error=0
+trap 'error=1' ERR
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+cd "$GIT_ROOT" || exit
+
+
+# Install required packages for h2o model artifacts test
```

Comment:
add a line here verifying jdk installed? as well as a comment about it?

Reply:
Yeah, I will comment on this.